exocom-server/CONTRIBUTING.md

```diff
@@ -11,6 +11,18 @@ The default port is 3100.
 More details around configuring the port [here](features/configuring-the-ports.feature).
 
 
+## Terminology
+
+- there are a number of __service types__, for example the "web" service, the "todo" service, etc
+- each service type has a number of instances. We call them __clients__ of Exocom to avoid the word "service" again.
+- each client has a unique name to identify it, e.g. "web #1"
+- the service configuration file defines which messages a service is allowed to send and receive.
+  This information is called __routing__
```

Comment:
In consistency with this, should the `messages` section of a service's `service.yml` be renamed to `routing`?